css/gallery.css

```diff
@@ -1,107 +1,130 @@
 /*class for the section of gallery*/
+.gallery-item::before {
+  content: " ";
+  display: block;
+  padding-top: 100%;
+}
 .gallery {
-  width: 635px;
-  height: 88px;
   font-family: Roboto;
   font-size: 16px;
   color: black;
-  padding: 60px 0;
+  padding-right: 20px;
```

Comment:
besides the right padding you are missing 20px padding on the left, 30px padding at the top and 40px padding at the bottom